feconf.py

```diff
@@ -193,6 +193,10 @@
 # tab.
 FEEDBACK_TAB_PAGE_SIZE = 20
 
+# The number of top unresolved answers which should be aggragated from all of
```

Comment:
"aggregated" is misspelled

Also...  "The number" --> "The maximum number", right?

Reply:
Yep. Done